diff --git a/node_modules/@antfu/utils/dist/index.d.mts b/node_modules/@antfu/utils/dist/index.d.mts
index 693e405..a37e208 100644
--- a/node_modules/@antfu/utils/dist/index.d.mts
+++ b/node_modules/@antfu/utils/dist/index.d.mts
@@ -354,9 +354,6 @@ declare class PInstance<T = any> extends Promise<Awaited<T>[]> {
     forEach(fn: (value: Awaited<T>, index: number) => void): Promise<void>;
     reduce<U>(fn: (previousValue: U, currentValue: Awaited<T>, currentIndex: number, array: Awaited<T>[]) => U, initialValue: U): Promise<U>;
     clear(): void;
-    then(fn?: () => PromiseLike<any>): Promise<any>;
-    catch(fn?: (err: unknown) => PromiseLike<any>): Promise<any>;
-    finally(fn?: () => void): Promise<Awaited<T>[]>;
 }
 /**
  * Utility for managing multiple promises.
