diff --git a/node_modules/@codemirror/language/dist/index.d.ts b/node_modules/@codemirror/language/dist/index.d.ts
index c40fcc0..a648cb3 100644
--- a/node_modules/@codemirror/language/dist/index.d.ts
+++ b/node_modules/@codemirror/language/dist/index.d.ts
@@ -725,7 +725,7 @@ declare class HighlightStyle implements Highlighter {
     */
     readonly module: StyleModule | null;
     readonly style: (tags: readonly Tag[]) => string | null;
-    readonly scope: ((type: NodeType) => boolean) | undefined;
+    readonly scope?: ((type: NodeType) => boolean);
     private constructor();
     /**
     Create a highlighter style that associates the given styles to
