diff --git a/node_modules/@pixi/color/lib/Color.d.ts b/node_modules/@pixi/color/lib/Color.d.ts
index 6b759ba..762ecb0 100644
--- a/node_modules/@pixi/color/lib/Color.d.ts
+++ b/node_modules/@pixi/color/lib/Color.d.ts
@@ -1,4 +1,4 @@
-import type { HslaColor, HslColor, HsvaColor, HsvColor, RgbaColor, RgbColor } from 'colord/types';
+import type { HslaColor, HslColor, HsvaColor, HsvColor, RgbaColor, RgbColor } from 'colord';
 /**
  * Value types for the constructor of {@link PIXI.Color}.
  * These types are extended from [colord](https://www.npmjs.com/package/colord) with some PixiJS-specific extensions.
