diff --git a/node_modules/@pixi/events/lib/FederatedEvent.d.ts b/node_modules/@pixi/events/lib/FederatedEvent.d.ts
index 171a26d..0c7e6c3 100644
--- a/node_modules/@pixi/events/lib/FederatedEvent.d.ts
+++ b/node_modules/@pixi/events/lib/FederatedEvent.d.ts
@@ -124,4 +124,9 @@ export declare class FederatedEvent<N extends UIEvent = UIEvent> implements UIEv
     readonly CAPTURING_PHASE = 1;
     readonly AT_TARGET = 2;
     readonly BUBBLING_PHASE = 3;
+
+    targetNode: Node | null;
+    win: Window;
+    doc: Document;
+    instanceOf<T>(type: { new(...data: any[]): T }): this is T;
 }
diff --git a/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts b/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts
index 89c2d5c..7da7ac0 100644
--- a/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts
+++ b/node_modules/@pixi/events/lib/FederatedPointerEvent.d.ts
@@ -62,4 +62,7 @@ export declare class FederatedPointerEvent extends FederatedMouseEvent implement
     detail: number;
     getCoalescedEvents(): PointerEvent[];
     getPredictedEvents(): PointerEvent[];
+
+    altitudeAngle: number;
+    azimuthAngle: number;
 }
