diff --git a/node_modules/@types/css-font-loading-module/index.d.ts b/node_modules/@types/css-font-loading-module/index.d.ts
index 728dd9c..e9cd295 100644
--- a/node_modules/@types/css-font-loading-module/index.d.ts
+++ b/node_modules/@types/css-font-loading-module/index.d.ts
@@ -85,7 +85,7 @@ declare global {
 
     var FontFace: {
         prototype: FontFace;
-        new(family: string, source: string | BinaryData, descriptors?: FontFaceDescriptors): FontFace;
+        new(family: string, source: string | BufferSource, descriptors?: FontFaceDescriptors): FontFace;
     };
 
     interface FontFaceSetLoadEvent extends Event {
