diff --git a/node_modules/electron/electron.d.ts b/node_modules/electron/electron.d.ts
index 87879e2..fe66c83 100644
--- a/node_modules/electron/electron.d.ts
+++ b/node_modules/electron/electron.d.ts
@@ -25562,9 +25562,10 @@ interface Document {
   createElement(tagName: 'webview'): Electron.WebviewTag;
 }
 
+type _ElectronProcessBase = import('node:events').InternalEventEmitter<import('node:process').ProcessEventMap>;
 
 declare namespace NodeJS {
-  interface Process extends NodeJS.EventEmitter {
+  interface Process extends _ElectronProcessBase {
 
     // Docs: https://electronjs.org/docs/api/process
 
@@ -25735,7 +25736,7 @@ declare namespace NodeJS {
      * `stderr`. Setting this to `true` will silence deprecation warnings. This
      * property is used instead of the `--no-deprecation` command line flag.
      */
-    noDeprecation: boolean;
+    noDeprecation?: boolean;
     /**
      * A `Electron.ParentPort` property if this is a `UtilityProcess` (or `null`
      * otherwise) allowing communication with the parent process.
