diff --git a/node_modules/obsidian/obsidian.d.ts b/node_modules/obsidian/obsidian.d.ts
index 73bb249..7a65311 100644
--- a/node_modules/obsidian/obsidian.d.ts
+++ b/node_modules/obsidian/obsidian.d.ts
@@ -6,6 +6,9 @@
 import { Extension, StateField } from '@codemirror/state';
 import { EditorView, ViewPlugin } from '@codemirror/view';
 import * as Moment from 'moment';
+import type { Mermaid } from 'mermaid' with { 'resolution-mode': 'import' };
+import type { default as pdfjsLib } from 'pdfjs-dist';
+import type { default as Prism } from 'prismjs';
 
 declare global {
     interface ObjectConstructor {
@@ -3378,7 +3381,7 @@ export function loadMathJax(): Promise<void>;
  * @see {@link https://mermaid.js.org/ Official Mermaid documentation}
  * @public
  */
-export function loadMermaid(): Promise<any>;
+export function loadMermaid(): Promise<Mermaid>;
 
 /**
  * Load PDF.js and return a promise to the global pdfjsLib object.
@@ -3386,7 +3389,7 @@ export function loadMermaid(): Promise<any>;
  * @see {@link https://mozilla.github.io/pdf.js/ Official PDF.js documentation}
  * @public
  */
-export function loadPdfJs(): Promise<any>;
+export function loadPdfJs(): Promise<typeof pdfjsLib>;
 
 /**
  * Load Prism.js and return a promise to the global Prism object.
@@ -3394,7 +3397,7 @@ export function loadPdfJs(): Promise<any>;
  * @see {@link https://prismjs.com/ Official Prism documentation}
  * @public
  */
-export function loadPrism(): Promise<any>;
+export function loadPrism(): Promise<typeof Prism>;
 
 /**
  * Location within a Markdown document
