diff --git a/node_modules/style-mod/package.json b/node_modules/style-mod/package.json
index 24b2706..d8b1c78 100644
--- a/node_modules/style-mod/package.json
+++ b/node_modules/style-mod/package.json
@@ -5,8 +5,14 @@
   "main": "dist/style-mod.cjs",
   "type": "module",
   "exports": {
-    "import": "./src/style-mod.js",
-    "require": "./dist/style-mod.cjs"
+    "import": {
+      "types": "./src/style-mod.d.ts",
+      "default": "./src/style-mod.js"
+    },
+    "require": {
+      "types": "./src/style-mod.d.cts",
+      "default": "./dist/style-mod.cjs"
+    }
   },
   "module": "src/style-mod.js",
   "types": "src/style-mod.d.ts",
diff --git a/node_modules/style-mod/src/style-mod.d.cts b/node_modules/style-mod/src/style-mod.d.cts
new file mode 100644
index 0000000..081f0f2
--- /dev/null
+++ b/node_modules/style-mod/src/style-mod.d.cts
@@ -0,0 +1,16 @@
+export class StyleModule {
+  constructor(spec: {[selector: string]: StyleSpec}, options?: {
+    finish?(sel: string): string
+  })
+  getRules(): string
+  static mount(
+    root: Document | ShadowRoot | DocumentOrShadowRoot,
+    module: StyleModule | ReadonlyArray<StyleModule>,
+    options?: {nonce?: string}
+  ): void
+  static newName(): string
+}
+
+export type StyleSpec = {
+  [propOrSelector: string]: string | number | StyleSpec | null
+}
